client/src/i18n.ts

```diff
@@ -17,7 +17,7 @@ function loadLocaleMessages(): LocaleMessages {
 }
 
 export default new VueI18n({
-    locale: process.env.VUE_APP_I18N_LOCALE || "en",
+    locale: localStorage.getItem("locale") || process.env.VUE_APP_I18N_FALLBACK_LOCALE || "en",
```

Comment:
Why did this become fallback locale ?

```suggestion
    locale: localStorage.getItem("locale") || process.env.VUE_APP_I18N_LOCALE || "en",
```